keyboards/kprepublic/jj50/rev2/keyboard.json

```
@@ -0,0 +1,142 @@
+{
+    "keyboard_name": "JJ50 rev2",
+    "manufacturer": "KPrepublic",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x4B50",
+        "pid": "0x0050",
+        "device_version": "2.0.0"
+    },
+    "matrix_pins": {
+        "cols": [
+            "B1",
+            "B0",
+            "F7",
+            "F0",
+            "C7",
+            "C6",
+            "B6",
+            "B5",
+            "B4",
+            "D7",
+            "D6",
+            "D4"
+        ],
+        "rows": ["D3", "D5", "E6", "F1", "F4"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
```

This is generally preferred to be enabled by default. If firmware size is a concern, you can disable some of the RGB effects, like `rgb_test`.
```suggestion
        "mousekey": true,
```

---

Updated. firmware size is still good 👍